benchmarks/index.md

```diff
@@ -104,14 +104,35 @@ key features. The problems are nontrivial and exhibit differing
 degrees of computational complexity but are not prohibitively
 computationally demanding. The outputs are defined to be easily
 comparable between different codes. The problems are also constructed
-test a simple, targeted aspect of either the numerical implementation
+to test a simple, targeted aspect of either the numerical implementation
 or the physics.
 
-The first set of Benchmark Problems involve diffusion of a solute and
-grain growth. Technically, they use the Cahn-Hilliard equation for a
-conserved order parameter, and coupled Cahn-Hilliard and Allen-Cahn
-equations for conserved and non-conserved order parameters. A second
-set has been tested on a workshop and Hackathon, and will be posted in
-the near future.
+The first set of Benchmark Problems, [BM1]({{ site.baseurl
+}}/benchmarks/benchmark1.ipynb/){:target="_blank"} and [BM2]({{
+site.baseurl }}/benchmarks/benchmark2.ipynb/){:target="_blank"},
+involve diffusion of a solute and grain growth. Technically, they use
+the Cahn-Hilliard equation for a conserved order parameter, and
+coupled Cahn-Hilliard and Allen-Cahn equations for conserved and
+non-conserved order parameters. The second set of Benchmark Problems,
```

Comment:
For each set of BM problems, might want to say what the intent was.  BM I:  to make sure your fundamentals are good.  BM II: length scales with dendritic growth, equilibrium linear elasticity.  BM III: extending to additional physics that are less commonly used for phase field.  BM IV: you said it already.

Reply:
I like it.